orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartStageHandler.kt

```diff
@@ -77,14 +77,14 @@ class StartStageHandler(
           queue.push(SkipStage(stage))
         } else {
           try {
+            // Set the startTime in case we throw an exception.
+            stage.startTime = clock.millis()
+
             stage.withAuth {
               stage.plan()
             }
 
-            stage.status = RUNNING
```

Comment:
what happened to this assignment? Pretty sure it's a) needed and b) cause of test failures

Reply:
Very needed, yes.

Reply:
You will also need to call `repository.storeStage(stage)` before doing the planning otherwise any exception thrown will result in the timestamp being lost.

Reply:
My bad. I'd moved this and a copy of store stage above the withAuth then decided not to and forgot to put it back.